.gitignore

```diff
@@ -43,6 +43,7 @@ bower.json
 # IntelliJ IDEA files
 .idea
 **.iml
+railways.cache
```

Comment:
railways is an intellij plugin to make dealing with routes easier. These are all links that take you to the controller method.

![screen shot 2016-08-30 at 9 08 43 am](https://cloud.githubusercontent.com/assets/4583474/18070474/65d3ab74-6e91-11e6-8ec7-4429831619d2.png)